poly-commitment/src/evaluation_proof.rs

```diff
@@ -1,3 +1,8 @@
+//! This module contains the implementation of the polynomial commitment scheme
+//! called the Inner Product Argument (IPA) as described in [Efficient
```

Comment:
I never understand why this module is called `evaluation_proof`... I guess it's something archaic before KZG was written?.. :thinking:

Reply:
Yes. We must rename it! It would be very nice as a first task!